packages/block-editor/src/components/block-types-list/index.native.js

```diff
@@ -81,7 +81,7 @@ export default function BlockTypesList( { name, items, onSelect, listProps } ) {
 			keyboardShouldPersistTaps="always"
 			numColumns={ numberOfColumns }
 			data={ items }
-			initialNumToRender={ 3 }
+			initialNumToRender={ 10 }
```

Comment:
Right now "3" rows are rendered in the FlatList when the inserter menu opens. "10" is just as hand wavy but now that it's a full screen render we might be able to remove this all together. At fullscreen there is only 1 or 2 rows that are off screen

These magic numbers should be parametrized regardless of this change set.